docs/io.cozy.accounts.md

```diff
@@ -143,31 +135,15 @@ in the `metadata` key of the relationship item.
 
 ```json
 {
+  "account_type": "freemobile",
   "auth": {
-    "accountName": "freemobile",
-    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp2pBEbXlZLWjiTzOGumGRomrF2LwlRn4Y8c=",
-    "folderPath": "/Administratif/Free Mobile",
+    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp...n4Y8c=",
     "login": "000000000",
-    "namePath": "Free Mobile",
-    "password": "*******"
-  }
-}
-```
-
-What we aim for:
-
-```json
-{
-  "auth": {
-    "credentials_encrypted": "aaDtwWSWsdpbbbbbbbbbbbbbbOzp2pBEbXlZLWjiTzOGumGRomrF2LwlRn4Y8c=",
-    "login": "000000000",
-    "password": "**********"
+    "password": "*******" // only when setting it first time
   },
-  "folderPath": "/Administratif/Free Mobile",
-  "label": "freemobile",
-  "namePath": "Free Mobile"
+  "defaultFolderPath": "/Administratif/Connector template/toto",
+  "state": "LOGIN_SUCCESS"
 }
-```
```

Comment:
I think you should not remove end code block here

Reply:
Good thinker !
Thanks